.github/workflows/ruby_tests.yml

```diff
@@ -60,6 +60,7 @@ jobs:
         run: |
           bundle exec rake app:db:create
           bundle exec rake app:db:migrate
+          bundle exec rake app:webpack:compile
```

Comment:
I'd probably rather drop the specs for the React pages. There is not much need, I just liked to have some kind of testing for them. But with JS testing, that would be redundant.

Reply:
Done :)